memtable/inlineskiplist.h

```diff
@@ -297,7 +297,7 @@ struct InlineSkipList<Comparator>::Node {
     return rv;
   }
 
-  const char* Key() const { return reinterpret_cast<const char*>(&next_[1]); }
+  const char* Key() const { return reinterpret_cast<const char*>(this + 1); }
```

Comment:
I prefer `&next_[2]`.

Reply:
Because the size of struct may align by 8.

Reply:
@zhangjinpeng1987 pointer type on x86-64 is 8 bytes, and here we only have pointer types in `Node`. I prefer better readability though.

@Little-Wallace maybe you can print `sizeof(Node)` for both cases and see?

Reply:
Another reason to use next[2] is to keep the original style.